airbyte-webapp/src/hooks/services/useClearbitReveal.ts

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from "react";
+
+import { ClearbitRevealProps } from "config";
+
+export const useClearbitReveal = () => {
+  const [reveal, setReveal] = useState<ClearbitRevealProps | null>(null);
+
+  useEffect(() => {
+    function onReveal() {
+      setReveal(window.reveal);
+    }
+
+    window.addEventListener("reveal", onReveal);
```

Comment:
I've looked into the GTM event, and I don't really see where at anytime a `reveal` event would be triggered?